package.json

```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "start": "NODE_ENV=production node ./src/server/server.babel.js",
     "dev": "NODE_ENV=development node ./src/server/server.babel.js",
-    "build": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\"",
+    "build:themes": "babel-node ./webpack/utils/buildThemeJSON.js",
```

Comment:
Oh that was a fun one!
I'm not sure I completely understand it, but here goes...
I was getting an error in the json loader where it was getting an empty string, which is what happens if you read an empty file. Maybe 20% of the time, only when the build folder existed.
I think that's because the server and client builds were both building that file. What I think was happening is that one would try to read the file before the other one built it, thanks to concurrently.